scripts/sql/create_databases.sql

```diff
@@ -1,5 +1,13 @@
 set global max_connections=500;
```

Comment:
This file can also be removed, was replaced by ota-mariadb-bootstrap.

Reply:
Deleted.